### PR TITLE
Support getPreviewer()

### PR DESCRIPTION
**Pull request by matsui54**
I will fix the import path and add doc if Shougo/ddu.vim#16 is marged.

### CODE REVIEW

File: denops/@ddu-kinds/file.ts [hunk 1]
```diff
@@ -3,15 +3,22 @@ import {
   Actions,
   BaseKind,
   DduItem,
+  PreviewContext,
+  Previewer,
   SourceOptions,
-} from "https://deno.land/x/ddu_vim@v1.5.0/types.ts";
-import { Denops, fn, op } from "https://deno.land/x/ddu_vim@v1.5.0/deps.ts";
+} from "../../../ddu.vim/denops/ddu/types.ts";
```
Comment (Shougo) [inline denops/@ddu-kinds/file.ts]: Why it uses relative path?   It should not.

File: denops/@ddu-kinds/file.ts [hunk 2]
```diff
@@ -3,15 +3,22 @@ import {
...
   isAbsolute,
   join,
   resolve,
 } from "https://deno.land/std@0.135.0/path/mod.ts";
+import {
+  Denops,
+  ensureObject,
+  fn,
+  op,
+} from "../../../ddu.vim/denops/ddu/deps.ts";
```
Comment (Shougo) [inline denops/@ddu-kinds/file.ts]: Why it uses relative path? It should not.

**Review by Shougo — CHANGES_REQUESTED** [review]
Please fix my commented code.

**Comment by Shougo** [comment]
The documentation update is needed for `PreviewOption`.

**Comment by matsui54** [comment]
Fixed the problems.

**Comment by Shougo** [comment]
I have merged it.  But `useBat` should specify preview command instead.

**Comment by matsui54** [comment]
Fixed. https://github.com/Shougo/ddu-kind-file/pull/4

**Pull request merged**